.github/workflows/dev-setup.yml

```diff
@@ -0,0 +1,60 @@
+name: Dev environment setup
+
+on:
+  schedule:
```

Comment:
I think we should run this workflow in PRs that change `dev/dev-env-setup.sh`.

Reply:
added